packages/ckeditor5-emoji/src/emojipicker.ts

```diff
@@ -432,6 +449,33 @@ export default class EmojiPicker extends Plugin {
 			} );
 		}
 	}
+
+	/**
+	 * Creates a div for emoji width testing purposes.
+	 */
+	private _createEmojiWidthTestingContainer(): HTMLDivElement {
+		const container = document.createElement( 'div' );
```

Comment:
To avoid processing, I would mark this element with a proper [aria attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden).